Assignment5/code.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+
+mysql -h mysql_db -uroot -psecret -e "create database baseball;"
+mysql -h mysql_db -uroot -psecret -D baseball < /data/baseball.sql
```

Comment:
This will create `baseball` in mysql on the first run but won't check if it already exists and then skip to the script. I can't remember if this will actually cause issues if `baseball` is already created and loaded with data.

Reply:
I ran the code with the baseball database already existing and it ran fine/exported tables, but I made the change anyway. It runs much faster the second time now. It must have been fully recreating the database on any subsequent runnings.